### ISSUE TITLE
V2: go get error when using gopkg.in

### ISSUE THREAD

**Issue by hodgesds** [urfave/cli#505]
When I run `go get -v gopkg.in/urfave/cli.v2/altsrc`

```sh
$ go get gopkg.in/urfave/cli.v2/altsrc
gopkg.in/urfave/cli.v2/altsrc
# gopkg.in/urfave/cli.v2/altsrc
../../../gopkg.in/urfave/cli.v2/altsrc/flag.go:80: f.EnvVars undefined (type *GenericFlag has no field or method EnvVars)
../../../gopkg.in/urfave/cli.v2/altsrc/flag.go:86: f.Names undefined (type *GenericFlag has no field or method Names)
../../../gopkg.in/urfave/cli.v2/altsrc/flag.go:118: f.EnvVars undefined (type *StringSliceFlag has no field or method EnvVars)
../../../gopkg.in/urfave/cli.v2/altsrc/flag.go:124: undefined: cli.NewStringSlice
../../../gopkg.in/urfave/cli.v2/altsrc/flag.go:125: f.Names undefined (type *StringSliceFlag has no field or method Names)
../../../gopkg.in/urfave/cli.v2/altsrc/flag.go:159: f.EnvVars undefined (type *IntSliceFlag has no field or method EnvVars)
../../../gopkg.in/urfave/cli.v2/altsrc/flag.go:165: undefined: cli.NewIntSlice
../../../gopkg.in/urfave/cli.v2/altsrc/flag.go:166: f.Names undefined (type *IntSliceFlag has no field or method Names)
../../../gopkg.in/urfave/cli.v2/altsrc/flag.go:200: f.EnvVars undefined (type *BoolFlag has no field or method EnvVars)
../../../gopkg.in/urfave/cli.v2/altsrc/flag.go:206: f.Names undefined (type *BoolFlag has no field or method Names)
../../../gopkg.in/urfave/cli.v2/altsrc/flag.go:206: too many errors
```

Running `go get -v gopkg.in/urfave/cli.v2` works as expected.

**Comment by hodgesds** [comment]
Seems to be related to #473

**Comment by jszwedko** [comment]
Fixed by #512 and #511

**Comment by hodgesds** [comment]
Thanks!
